modules/contact-form/grunion-contact-form.php

```diff
@@ -749,7 +749,7 @@ public function map_parsed_field_contents_of_post_to_field_names( $parsed_post_c
 	 * @return array $exporters An array of personal data exporters.
 	 */
 	public function register_personal_data_exporter( $exporters ) {
-		$exporters[] = array(
+		$exporters['jetpack-contact-form'] = array(
```

Comment:
Do you think it makes sense to name this `jetpack-feedback` instead of contact form?

Reply:
I think either one is probably fine. I chose `contact-form` because that's the name of the component in the code, but since the exporter and eraser specifically relate to the `feedback` post type, that seems equally valid. Up to you.

Thanks!

Reply:
Cool, let's go with `jetpack-feedback` please. This LGTM in every other way. 💯